imports/plugins/core/orders/client/components/orderActions.js

```diff
@@ -1,112 +1,168 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import classnames from "classnames/dedupe";
-import { Button, DropDownMenu, MenuItem } from "@reactioncommerce/reaction-ui";
+import { Components } from "@reactioncommerce/reaction-components";
 
 class OrderActions extends Component {
   static propTypes = {
-    className: PropTypes.string,
+    className: PropTypes.object,
```

Comment:
This variable name sounds like it should be a string but it appears it's not now. Can we give it a more descriptive name?